debian/copyright

```diff
@@ -1,16 +1,22 @@
-This package was debianized by Michael Prokop <mika@grml.org> on
-Mon, 23 May 2005 23:39:39 +0200.
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Source: https://github.com/grml/grml-x/
 
-It was downloaded from http://grml.org/
-
-Copyright Holder: Michael Prokop <mika@grml.org>
-
-License:
-
- This software is copyright (c) 2005 by Michael Prokop.
-
- You are free to distribute this software under the terms of
- the GNU General Public License.
-
- On Debian GNU/Linux systems, the complete text of the GNU General
- Public License can be found in `/usr/share/common-licenses/GPL'.
+Files: *
+Copyright: 2007 Alexander Wirt <formorer@debian.org>
+           2010, 2011, 2025 Christian Hofstaedtler <zeha@debian.org>
+           2013 Evgeni Golov <evgeni@debian.org>
+           2005-2025 Michael Prokop <mika@grml.org>
+           2010 Michael Stapelberg <michael@stapelberg.de>
```

Comment:
I'm a proponent of crediting folks for their contributions, but Michael Stapelberg had a single word change, Evgeni a single line change and Alex a trivial 8 line change that's no longer present as such, so I wouldn't add them as copyright holders here :)

Reply:
ok, i just looked at Author: and Committer: from git log. will remove them then

Reply:
done!